### ISSUE TITLE
jsonnet-bundler not installing on ubuntu-18.04

### ISSUE THREAD

**Issue by bwillcox** [jsonnet-bundler/jsonnet-bundler#23]
The following go command was documented [here](https://github.com/coreos/prometheus-operator/blob/master/contrib/kube-prometheus/README.md#customizing-kube-prometheus) for installing jsonnet-bundler; however, the go command returns no error indication, but the jb command appears unavailable despite the apparent success. Any ideas?

ubuntu % go get -v github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
ubuntu % pwd
/home/bwillcox/jsonnet-bundler
ubuntu % 
ubuntu % go get -v github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
ubuntu % jb
jb: command not found
ubuntu % cat /etc/os-release 
NAME="Ubuntu"
VERSION="18.04.1 LTS (Bionic Beaver)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 18.04.1 LTS"
VERSION_ID="18.04"
HOME_URL="https://www.ubuntu.com/"
SUPPORT_URL="https://help.ubuntu.com/"
BUG_REPORT_URL="https://bugs.launchpad.net/ubuntu/"
PRIVACY_POLICY_URL="https://www.ubuntu.com/legal/terms-and-policies/privacy-policy"
VERSION_CODENAME=bionic
UBUNTU_CODENAME=bionic
ubuntu % go version
go version go1.10.4 linux/amd64

**Comment by metalmatze** [comment]
If the `go get` command doesn't finish with an error everything should have worked properly. Do you have `$GOPATH/bin` in your `$PATH` though? That is mandatory for `jb` to work everywhere on your system.
You can find out your `$GOPATH` by running `go env`.

**Comment by bwillcox** [comment]
yep adding $GOPATH/bin to my PATH fixed it; thank you!

**Comment by D1nay** [comment]
Yep, adding $GOPATH/bin to the PATH solved it for me as well! Thanks for mentioning the commands to look up the environment variables for Go.
